### PR TITLE
MTS and ITE configuration changes

### PR DESCRIPTION
**Pull request by DarrenInwood**
Configuration change to suit new ITE endpoint, tested and working on CWP.

Have also changed MTS settings to make it easier for users to get MTS integrations working using self-signed SSL certificates.

### CODE REVIEW

**Comment by tractorcow** [comment]
@madmatt please review and merge if you're happy with this.

File: templates/simplesaml-configuration/authsources.php [hunk 1]
```diff
@@ -107,7 +107,7 @@
         'capture_peer_cert' => true,
         'allow_self_signed' => false,
         'verify_depth' => 5,
-        'peer_name' => 'as.ite.logon.realme.govt.nz',
```
Comment (madmatt) [inline templates/simplesaml-configuration/authsources.php]: Hey @DarrenInwood, I'm happy with the other changes, but are you able to remove this ITE change?

ITE changes also require a different certificate to be added (with a different filename) and for the `saml20-idp-remote.php` file to be updated with the new service and artifact resolution URLs, so we may as well handle it all as one separate pull request if that makes sense?

**Comment by madmatt** [comment]
Hey Darren,

I've made one comment if you could please review and it's all good to be merged in, thanks for that :)

**Comment by madmatt** [comment]
FYI, I've made a new pull request for the updates to ITE and Prod changes, this will require a version bump to 0.9.1, see here: #15

**Pull request closed without merging**